torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6846,16 +6847,25 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            aten_name='linalg_eigh',
            dtypes=floating_and_complex_types(),
            check_batched_gradgrad=False,
+           supports_forward_ad=True,
            sample_inputs_func=sample_inputs_linalg_eigh,
            gradcheck_wrapper=gradcheck_wrapper_hermitian_input,
-           decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack]),
+           decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],
+           skips=(
+               # Gradcheck for complex hangs for this function
+               SkipInfo('TestGradients', 'test_forward_mode_AD', dtypes=complex_types()),),
```

Comment:
Do we have an issue open tracking this? Do we want one if not?

Reply:
There is no open issue about it. I added a NotImplementedError for complex inputs for now, we'll enable it later.